lib/client.js

```diff
@@ -48,6 +48,11 @@ function Client (broker, conn, req) {
 
   this.connDetails = null
```

Comment:
this.connDetails can be removed if not used.

Reply:
It is used [here](https://github.com/moscajs/aedes/blob/master/lib/protocol-decoder.js#L116)

And @mcollina suggested @getlarge to init it to null in client in his PR